### ISSUE TITLE
Mobx: CkanCatalogGroup making requests on open/close of group

### ISSUE THREAD

**Issue by rowanwins** [TerriaJS/terriajs#4495]
Currently the `CkanCatalogGroup` is calling the CKAN API on every open and close of the group in the catalog. this isn't ideal.

**Comment by rowanwins** [comment]
Need to modify `forceLoadMetadata` something like this I think...

````
  protected forceLoadMetadata(): Promise<void> {
    const portalStratum = <ArcGisPortalStratum | undefined>(
      this.strata.get(ArcGisPortalStratum.stratumName)
    );
    if (!portalStratum) {
      return ArcGisPortalStratum.load(this).then(stratum => {
        if (stratum === undefined) return;
        runInAction(() => {
          this.strata.set(ArcGisPortalStratum.stratumName, stratum);
        });
      });
    } else {
      return Promise.resolve()
    }
  }
````

**Comment by rowanwins** [comment]
Fixed by #4501
